nvflare/app_common/ccwf/ccwf_job.py

```diff
@@ -249,7 +262,10 @@ def add_swarm(
             wait_time_after_min_resps_received=client_config.wait_time_after_min_resps_received,
         )
         self.to_clients(client_controller, tasks=["swarm_*"])
-        self.to_clients(client_config.executor, tasks=["train", "validate", "submit_model"])
+        if not self.executor:
+            # We add the executor only if it's not added yet.
```

Comment:
If the executor is already added, would it be clearer to raise an error? Or the current way of disregarding any additional client_config.executors is good enough

Reply:
The user could call add_swarm and then add_cyclic using the same executor in the client_config. Otherwise the user would have to set the "executor" to None in client_config of the 2nd call, which is not allowed, and would make it not reusable.